imports/plugins/core/cart/server/no-meteor/util/addCartItems.js

```diff
@@ -133,7 +134,7 @@ export default async function addCartItems(context, currentItems, inputItems, op
       shopId: catalogProduct.shopId,
       // Subtotal will be kept updated by event handler watching for catalog changes.
       subtotal: {
-        amount: variantPriceInfo.price * quantity,
+        amount: +toFixed(variantPriceInfo.price * quantity, 3),
```

Comment:
Am I reading this right that we're fixing to 3 decimal places? If so, why 3?

Reply:
Based of @aldeed's previous work on related issues, some currency codes go up to 3 decimals, so we need to accommodate those.

Algeria, Bahrain, Iraq, Jordan, Kuwait, Libya and Tunisia